### PR TITLE
podman network exists

### PR DESCRIPTION
**Pull request by Luap99**
Add podman network exists command with remote support.

### CODE REVIEW

File: docs/source/markdown/podman-network-exists.1.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+$ podman network exists webbackend
+$ echo $?
+1
+$
+```
+## SEE ALSO
+podman(1), podman-network-create(1), podman-network-rm(1)
+
+## HISTORY
+Januar 2021, Originally compiled by Paul Holzinger <paul.holzinger@web.de>
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-network-exists.1.md:43]:
```suggestion
January 2021, Originally compiled by Paul Holzinger <paul.holzinger@web.de>
```

Comment (Luap99) [reply docs/source/markdown/podman-network-exists.1.md:43]: fixed

**Comment by baude** [comment]
sharp work @Luap99

**Comment by baude** [comment]
LGTM

**Review by rhatdan — APPROVED** [review]
/lgtm

**Pull request merged**